airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/schemas/time_off_requests.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "employeeId": {
+      "type": ["null", "string"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "start": {
+      "type": ["null", "string"],
+      "default": ""
+    },
+    "end": {
+      "type": ["null", "string"],
+      "default": ""
+    },
+    "created": {
+      "type": ["null", "string"],
+      "default": ""
+    },
+    "status": {
+      "type": ["null", "object"]
+    },
+    "dates": {
+      "type": ["null", "string"],
+      "default": {}
+    },
+    "type": {
+      "type": ["null", "object"]
+    },
+    "amount": {
+      "type": ["null", "object"]
+    },
+    "actions": {
+      "type": ["null", "object"]
+    },
+    "notes": {
+      "type": ["null", "object"]
```

Comment:
```suggestion
      "type": ["null", "object"],
      "additionalProperties": true,
      "properties": {}
```

Reply:
You need to make all objects like this. It is a requirement from the Airbyte cDK